apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/GetUserResponseNode.cs

```diff
@@ -13,21 +13,25 @@
 // limitations under the License.
 
 using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
-using Energinet.DataHub.WebApi.GraphQL.Resolvers;
 
-namespace Energinet.DataHub.WebApi.GraphQL.Types.User;
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.User;
 
-public class GetUserResponseType : ObjectType<GetUserResponse>
+[ObjectType<GetUserResponse>]
+public static partial class GetUserResponseNode
```

Comment:
suggestion: the same with this class. I think this class should be named `UserNode` and a descriptor should change the name of `GetUserResponse` to just `User`:

```cs
descriptor.Name("User");
```